web-app/src/components/Commentary/data.js

```diff
@@ -0,0 +1,97 @@
+import { useContext, useState } from "react";
+import { makeStyles } from '@material-ui/core/styles';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import IconButton from '@material-ui/core/IconButton';
+import { Commentary } from "../../contexts/commentary"
+import CustomizedDialogs from "./Dialog";
+import { CommonContext } from "../../contexts/Common";
+
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+  },
+  table: {
+    minWidth: 650,
+  },
+  heading: {
+    fontSize: theme.typography.pxToRem(15),
+    flexBasis: '33.33%',
+    flexShrink: 0,
+    textTransform:'capitalize'
+  },
+  secondaryHeading: {
+    fontSize: theme.typography.pxToRem(15),
+    color: theme.palette.text.secondary,
+  },
+}));
+
+function Data() {
+   const { isLoading,error,data:commentaries } =  useContext(Commentary)
+   if (isLoading) return 'Loading...'
+ 
+   if (error) return  error.message  
```

Comment:
This also will work but instead of this we can use commentary.languageCode, that would be more appropriate, as this is the key for the language accordian.